### ISSUE TITLE
cytof_cluster does not contain Rphenograph_k argument 

### ISSUE THREAD

**Issue by classycammy** [JinmiaoChenLab/cytofkit#37]
Hi, I believe that the function of "cytof_cluster" does not contain Rphenograph_k. When I run in R, I received this error:

> Cluster_PhenoGraph <- cytof_cluster(xdata= combined_data_transformed_40K, method="Rphenograph", Rphenograph_k=45)
Error in cytof_cluster(xdata = combined_data_transformed_40K, method = "Rphenograph",  : 
  unused argument (Rphenograph_k = 45)

Could you please advise? Thank you.

**Comment by SamGG** [comment]
Hi,
Here is a dummy code that shows no problem with the latest version of cytofkit. Could you report the version you are running?
```
packageVersion("cytofkit")
set.seed(0); cytof_cluster(xdata = matrix(rnorm(1000), 100, 10), method = "Rphenograph", Rphenograph_k = 20)
```
![2018-05-14_084012](https://user-images.githubusercontent.com/3333348/39981859-aa1e73a8-5752-11e8-8f73-989ca0c12ae2.png)

**Comment by SamGG** [comment]
Quite old in fact. You found the problem: the parameter is not available in this version whereas the documentation reflects the use of the latest version. I think you should upgrade R to 3.5.0 and bioconductor 3.7.
If you don't want to change your install, you could try to install cytofkit using [devtools and github](https://github.com/JinmiaoChenLab/cytofkit#2-install-cytofkit-package), but the success of such an install relies on the fact that cytofkit are resolved or could be resolved in the current R environment. Try it!

**Comment by classycammy** [comment]
Right. Thank you so much, Sam. Appreciate it tons. It is good to know that it is due to the version. I will upgrade to the version you have recommended.

**Comment by SamGG** [comment]
If this issue is solved, could you close it?
Best.
